src/Card/CardSection/CardSectionContent/index.js

```diff
@@ -33,11 +21,13 @@ export const StyledCardSectionContent = styled.div`
       : `0px solid ${theme.orbit.paletteCloudNormal}`};
   padding-top: ${({ theme, expandable, expanded, visible }) =>
     hasPaddingTop({ expandable, expanded, visible }) && theme.orbit.spaceLarge};
-  transition: max-height ${({ theme }) => theme.orbit.durationFast} linear,
-    padding ${({ theme }) => theme.orbit.durationFast} linear,
+  transition: padding ${({ theme }) => theme.orbit.durationFast} linear,
     border-top ${({ theme }) => theme.orbit.durationFast} linear;
-  max-height: ${getMaxHeight};
   overflow: hidden;
+
+  ${StyledSlide} {
```

Comment:
Really necessary, can be this done by `Slide` component itself? 🤔

Reply:
Yes, it is case if CardContent is default visible.

This is little bit different behaviour then obviously.

![image](https://user-images.githubusercontent.com/7850592/54032675-9a497400-41b2-11e9-9bb6-da3601d6fc8e.png)

Reply:
Kay